src/parser.ts

```diff
@@ -220,11 +344,27 @@ export const parseCedarDocEntities = (
   if (visitEntity === undefined) {
     const cachedItem = entityCache[entitiesDoc.uri.toString()];
     if (cachedItem && cachedItem.version === entitiesDoc.version) {
-      // console.log("parseCedarDocEntities (cached)");
+      // console.log("parseCedarEntitiesDoc (cached)");
       return cachedItem;
     }
   }
 
+  let UID = 'uid';
+  let TYPE = 'type';
+  let ID = 'id';
+  let ENTITY = '__entity';
+  let ATTRS = 'attrs';
+  let PARENTS = 'parents';
+
+  // Amazon Verified Permissions entities format has a similar structure but different property names
```

Comment:
There are other differences too, viz explicitly-typed attribute values.  Nonetheless the two formats can still be automatically converted back and forth, and the cedarpolicy.com playground performs this conversion as you toggle between the two formats.

Reply:
Wasn't meaning to imply that we needed that full support in this PR, just that the comment seemed a little misleading